public/app/plugins/datasource/prometheus/components/PromExemplarField.tsx

```diff
@@ -1,21 +1,67 @@
-import { InlineField, InlineSwitch } from '@grafana/ui';
-import React from 'react';
+import { GrafanaTheme } from '@grafana/data';
+import { FetchError } from '@grafana/runtime';
+import { IconButton, InlineLabel, Tooltip, useStyles } from '@grafana/ui';
+import { css, cx } from 'emotion';
+import React, { useEffect, useState } from 'react';
+import { PrometheusDatasource } from '../datasource';
 import { PromQuery } from '../types';
 
 interface Props {
   query: PromQuery;
   onChange: (value: PromQuery) => void;
+  datasource: PrometheusDatasource;
 }
 
-const onExemplarsChange = ({ query, onChange }: Props) => (e: React.ChangeEvent<HTMLInputElement>) => {
-  const exemplar = e.target.checked;
-  onChange({ ...query, exemplar });
-};
-
 export function PromExemplarField(props: Props) {
+  const [error, setError] = useState<FetchError>();
+  const styles = useStyles(getStyles);
+
+  useEffect(() => {
+    const subscription = props.datasource.exemplarErrors.subscribe((err) => {
+      setError(err);
+    });
+    return () => {
+      subscription.unsubscribe();
+    };
+  }, [props]);
+
+  const iconButtonStyles = cx(
+    {
+      [styles.activeIcon]: !!props.query.exemplar,
+    },
+    styles.eyeIcon
+  );
+
   return (
-    <InlineField label="Exemplars" labelWidth="auto">
-      <InlineSwitch label="Exemplars" value={!!props.query.exemplar} onChange={onExemplarsChange(props)} />
-    </InlineField>
+    <InlineLabel width="auto">
+      <Tooltip content={!!error ? 'Exemplars are not supported in this version of prometheus.' : ''}>
+        <div className={styles.iconWrapper}>
+          Exemplars
+          <IconButton
+            name="eye"
```

Comment:
I would do here also `tooltip={!!props.query.exemplar ?  'Disable query with exemplars' : 'Enable query with exemplars'}`. Not sure about exact copy, but I would add tooltip for this as well.